deploy/20_ocm-agent-operator.ClusterRole.yaml

```diff
@@ -45,4 +44,11 @@ rules:
     verbs:
       - get
       - list
-      - watch
\ No newline at end of file
+      - watch
+  - apiGroups:
```

Comment:
I would like investigate this more, it seems wrong to broaden our RBAC given the only clusterrole it cares about is in the OAO namespace. We will undoubtedly face this issue again in future migrations so we should seek to understand what the underlying issue is.